rhel8/profiles/C2S.profile

```diff
@@ -0,0 +1,245 @@
+documentation_complete: true
+
+title: 'C2S for Red Hat Enterprise Linux 7'
```

Comment:
C2S reflects the baseline for CIA's Commercial Cloud Services (C2S). There isn't a C2S baseline yet and likely won't be for some time. For now, there shouldn't be a RHEL8 C2S profile.